snippets/csharp/System.Linq/Queryable/AggregateTSource/queryable.cs

```diff
@@ -1191,14 +1203,16 @@ public static void GroupJoinEx1()
         static void IntersectEx1()
         {
             // <Snippet41>
-            int[] id1 = { 44, 26, 92, 30, 71, 38 };
-            int[] id2 = { 39, 59, 83, 47, 26, 4, 30 };
+            int[] id1 = [44, 26, 92, 30, 71, 38];
+            int[] id2 = [39, 59, 83, 47, 26, 4, 30];
 
             // Get the numbers that occur in both arrays (id1 and id2).
-            IEnumerable<int> both = id1.AsQueryable().Intersect(id2);
+            IEnumerable<int> both = id1.Intersect(id2);
```

Comment:
Revert this. These samples are using the `Queryable` implementation intentionally.